File to verify distance assumption

tedim52 opened the pull request:
-File to verify distance assumption with euclidean distance function
-Assumption: Low distance value indicates similar Googlers.
-Currently uses a euclidean distance function on all 7 attributes.

tedim52 committed:
File to verify distance assumption

tedim52 committed:
Small change to file

DeanAlvarez commented on verifying_distance.py:
```diff
@@ -0,0 +1,71 @@
+
+        googler_one_attributes = googler_data.iloc[user_one_id].values[1:6 + 1]
+        googler_two_attributes = googler_data.iloc[user_two_id].values[1:6 + 1]
+        l2 = l2_distance(googler_one_attributes, googler_two_attributes, 6)
```
you could call this distance instead of l2 to further how modular the code is

DeanAlvarez reviewed (approved):
LGTM

DeanAlvarez commented on verifying_distance.py:
```diff
@@ -0,0 +1,71 @@
+
+print(googler_data.head())
+
+#Create distance function to measure l2 distance between two Googlers
```
maybe define l2 distance here?

tedim52 committed:
Delete unecessary setup.sh, create testing script for script

tedim52 committed:
Added testing file

tedim52 committed:
Merge branch 'master' of https://github.com/kpatterson01/step-capstone-2020

tedim52 committed:
Merge branch 'master' into tedi

tedim52 committed:
Add testing files and test cases, modify distance function

tedim52 committed:
Change name of 3rd column to distance

tedim52 commented:
-Utilized normalize function from normalizing_data.py
-Allowed function to be taken is input in sample so any distance function we write can be sampled
-Added unit test file for distance_validation.py using unittest module
-Added unit tests for euclidean aka l2 norm distance function

tedim52 commented:
-Changed l2 distance function to use built in numpy l2 distance function

tedim52 committed:
Renamed files verifying_distance.py to distance_validation.py

meritus commented on distance_validation.py:
```diff
@@ -0,0 +1,70 @@
+import pandas as pd
+import normalizing_data
+
+#Take in user attribute data as DataFrame
```
Typically, if you ran the linter it would say put a space before the start of comments, like above.

meritus reviewed (approved):
LGTM

High level, non-blocking question. Potentially it would be good to describe on the file level what everything is use for **in addition** to what it does.

tedim52 committed:
Merge branch 'master' into tedi
